appengine/standard/ndb/schema_update/add_entities_handler.py

```diff
@@ -0,0 +1,32 @@
+import os
+import time
+
+import jinja2
+import webapp2
+
+import update_schema
```

Comment:
I don't think this module should import update schema at all. It just should just use models_v1 directly.